NDTensors/src/lib/Sectors/src/symmetry_style.jl

```diff
@@ -0,0 +1,31 @@
+# This file defines SymmetryStyle, a trait to distinguish abelian groups, non-abelian groups
+# and non-group fusion categories.
+
+using BlockArrays
+
+using NDTensors.LabelledNumbers
+using NDTensors.GradedAxes
```

Comment:
Are these needed? If so, please list which functions/types are being used explicitly, i.e. `using BlockArrays: ...`.

Reply:
These lines were indeed wrong and my intention was `using BlockArrays: BlockArrays.` I changed to explicit import.